### DISCUSSION TITLE
PHP  8.1 loads PHP v8.1.2 explicitly whereas i need to get php v8.1.0 to match dependencies in my drupal build.

### DISCUSSION THREAD

**Discussion by ahsanra** [ddev/ddev#5321] [General]
I need to update or downgrade php version inside my container and don;t know how to do that, can someone please guide me?

(Config = PHP 8.1) loads PHP v8.1.2 explicitly whereas i need to get php v8.1.0 to match dependencies in my drupal build. 
How i down grade inside the drupal container?

**Comment by tyler36** [comment]
Unfortunately, it is not possible. 

> You can only specify the major version (7.3), not a minor version (7.3.2), from those explicitly available.
https://ddev.readthedocs.io/en/latest/users/configuration/config/#php_version

**Comment by rfay** [comment]
There may be a version of DDEV that had 8.1.0, but I think you're probably in the wrong place, because any composer spec that specifies a patch version is a very unusual thing, and you won't be able to build anywhere, it's not just a DDEV problem.
